src/dbus/server/instropect.xml

```diff
@@ -182,7 +182,117 @@
+    </property>
+
+    <property name="ChannelMonitorSampleCount" type="u" access="read">
+      <annotation name="org.freedesktop.DBus.Property.EmitsChangedSignal" value="false"/>
+    </property>
+
+    <!--
+      struct {
+        uint8_t  mChannel;
+        uint16_t mOccupancy;
+      }
+    -->
+    <property name="ChannelMonitorChannelQualityMap" type="a(yq)" access="read">
+      <annotation name="org.freedesktop.DBus.Property.EmitsChangedSignal" value="false"/>
+    </property>
+
+    <!--
+      struct {
+        uint64_t mExtAddress;         ///< IEEE 802.15.4 Extended Address
+        uint32_t mTimeout;            ///< Timeout
+        uint32_t mAge;                ///< Time last heard
+        uint16_t mRloc16;             ///< RLOC16
+        uint16_t mChildId;            ///< Child ID
+        uint8_t  mNetworkDataVersion; ///< Network Data Version
+        uint8_t  mLinkQualityIn;      ///< Link Quality In
+        int8_t   mAverageRssi;        ///< Average RSSI
+        int8_t   mLastRssi;           ///< Last observed RSSI
+        uint16_t mFrameErrorRate;     ///< Frame error rate (0xffff->100%). Requires error tracking feature.
+        uint16_t mMessageErrorRate;   ///< (IPv6) msg error rate (0xffff->100%). Requires error tracking feature.
+        bool     mRxOnWhenIdle;       ///< rx-on-when-idle
+        bool     mSecureDataRequest;  ///< Secure Data Requests
+        bool     mFullThreadDevice;   ///< Full Thread Device
+        bool     mFullNetworkData;    ///< Full Network Data
+        bool     mIsStateRestoring;   ///< Is in restoring state
+      }
+    -->
+    <property name="ChildTable" type="a(tuuqqyyyyqqbbbbb)" access="read">
+      <annotation name="org.freedesktop.DBus.Property.EmitsChangedSignal" value="false"/>
+    </property>
+
+    <!--
+      struct {
+        uint64_t mExtAddress;        ///< IEEE 802.15.4 Extended Address
+        uint32_t mAge;               ///< Time last heard
+        uint16_t mRloc16;            ///< RLOC16
+        uint32_t mLinkFrameCounter;  ///< Link Frame Counter
+        uint32_t mMleFrameCounter;   ///< MLE Frame Counter
+        uint8_t  mLinkQualityIn;     ///< Link Quality In
+        int8_t   mAverageRssi;       ///< Average RSSI
+        int8_t   mLastRssi;          ///< Last observed RSSI
+        uint16_t mFrameErrorRate;    ///< Frame error rate (0xffff->100%). Requires error tracking feature.
+        uint16_t mMessageErrorRate;  ///< (IPv6) msg error rate (0xffff->100%). Requires error tracking feature.
+        bool     mRxOnWhenIdle;      ///< rx-on-when-idle
+        bool     mSecureDataRequest; ///< Secure Data Requests
+        bool     mFullThreadDevice;  ///< Full Thread Device
+        bool     mFullNetworkData;   ///< Full Network Data
+        bool     mIsChild;           ///< Is the neighbor a child
+      }
+    -->
+    <property name="NeighborTable" type="a(tuquuyyyqqbbbb)" access="read">
```

Comment:
5 boolean fields in the end of the structure, however here only `b`x4?

Reply:
seems also inconsistent with the NeighborInfo here
https://github.com/openthread/ot-br-posix/blob/810fdae2693d314c3fa0e8120db186d67d25d991/src/dbus/common/dbus_message_helper.hpp#L120-L125

Reply:
Thanks for pointing out. I've also updated the unit tests to check these type signatures.